lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -12,15 +12,18 @@ describe('LMSGrader', () => {
       LISOutcomeServiceUrl: '',
     },
     {
-      userid: 'student1',
+      userid: 'student2',
```

Comment:
I'm wondering if it might be a good idea to use `userid`s that look like actual `userid`s to avoid further confusing future devs.

Reply:
yep, good idea